packages/ui/src/components/SelectableCard/index.tsx

```diff
@@ -352,7 +352,9 @@ export const SelectableCard = forwardRef(
         if (tooltip) {
           return (
             <Stack flex={1}>
-              <Tooltip text={tooltip}>{subChildren}</Tooltip>
+              <Tooltip text={tooltip} containerFullHeight>
+                {subChildren}
+              </Tooltip>
             </Stack>
```

Comment:
Doesn't it work if instead you do that to the parent of the tooltip? Maybe using `flex="1 1 auto"` might do something.